### ISSUE TITLE
Issue in Viewfinder.visibleGameSizze while doing the Klondike tutorial

### ISSUE THREAD

**Issue by MasterKavi-24** [flame-engine/flame#2797]
# Current bug behavior
```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following assertion was thrown building FutureBuilder<void>(dirty, state:
_FutureBuilderState<void>#98053):
zoom level must be positive: 0.0
'package:flame/src/camera/viewfinder.dart':
Failed assertion: line 47 pos 12: 'value > 0'

The relevant error-causing widget was:
  GameWidget<KlondikeGame>
  GameWidget:file:///C:/Users/srika/Desktop/World/Flutter/flame_1/lib/main.dart:8:10

When the exception was thrown, this was the stack:
#2      Viewfinder.zoom= (package:flame/src/camera/viewfinder.dart:47:12)
#3      Viewfinder._initZoom (package:flame/src/camera/viewfinder.dart:166:7)
#4      Viewfinder.visibleGameSize= (package:flame/src/camera/viewfinder.dart:133:7)
#5      KlondikeGame.onLoad (package:flame_1/klondike_game.dart:52:20)
<asynchronous suspension>
#6      GameWidgetState.loaderFuture.<anonymous closure> (package:flame/src/game/game_widget/game_widget.dart:196:11)
<asynchronous suspension>
#7      _FutureBuilderState._subscribe.<anonymous closure> (package:flutter/src/widgets/async.dart:624:33)
<asynchronous suspension>
(elided 2 frames from class _AssertionError)

════════════════════════════════════════════════════════════════════════════════════════════════════
```

# Steps to reproduce
I followed, and even copied the same code from the [2. Scaffolding - Flame (Klondike)](https://docs.flame-engine.org/latest/tutorials/klondike/step2.html)

# Flutter doctor output
```
C:\Windows\system32>flutter doctor -v
[√] Flutter (Channel stable, 3.13.6, on Microsoft Windows [Version 10.0.19045.3448], locale en-IN)
    • Flutter version 3.13.6 on channel stable at C:\Users\srika\Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ead455963c (10 days ago), 2023-09-26 18:28:17 -0700
    • Engine revision a794cf2681
    • Dart version 3.1.3
    • DevTools version 2.25.0

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Program Files\Android\Sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-10027231)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.7.4)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.7.34031.279
    X The current Visual Studio installation is incomplete. Please reinstall Visual Studio.

[√] Android Studio (version 2022.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-10027231)

[√] VS Code, 64-bit edition (version 1.82.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.56.0

[√] Connected device (4 available)
    • M2101K7BI (mobile) • IRGEH6ZLDUTGT86H • android-arm64  • Android 12 (API 31)
    • Windows (desktop)  • windows          • windows-x64    • Microsoft Windows [Version 10.0.19045.3448]
    • Chrome (web)       • chrome           • web-javascript • Google Chrome 117.0.5938.134
    • Edge (web)         • edge             • web-javascript • Microsoft Edge 117.0.2045.47

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

# More environment information
* Flame version: ^1.9.1
* Platform affected: android

**Comment by spydon** [comment]
Ah yes, that needs to be fixed, try this meanwhile:
```dart
    final camera = CameraComponent(world: world) 
      ..viewfinder.position = Vector2(cardWidth * 3.5 + cardGap * 4, 0)
      ..viewfinder.anchor = Anchor.topCenter;
    add(camera);
    camera.viewfinder.visibleGameSize =
          Vector2(cardWidth * 7 + cardGap * 8, 4 * cardHeight + 3 * cardGap)
```

**Comment by qmffkem** [comment]
hello @spydon,
Thank you for your quick fix; I was also one of the person following the tutorial and encountered the same issue with zoom level = 0.
After checking the relavent PR, it looks like the fix has already been merged into the main branch(#2813), but it's not updated on the actual website; is it because the webpage is not most up-to-date version? just curious about how it happened.

**Comment by spydon** [comment]
> hello @spydon,
> Thank you for your quick fix; I was also one of the person following the tutorial and encountered the same issue with zoom level = 0.
> After checking the relavent PR, it looks like the fix has already been merged into the main branch(#2813), but it's not updated on the actual website; is it because the webpage is not most up-to-date version? just curious about how it happened.
> 

It is updated, but you have to go to the `main` version of the docs, you can choose it in the drop-down in the top right. The `latest` is updated when there is a new release. :)

**Comment by qmffkem** [comment]
🤦 to my self.. Thanks, @spydon!
